### PR TITLE
added create account requests with templates for requests

### PR DESCRIPTION
**Pull request by zhao19991111**
- Added request function for creating account with the following format
- POST "/signUp"
- body:
- {
-     firstName: String,
-     lastName: String,
-     email: String,
-     password: String,
-     isTutor: Boolean
- }

### CODE REVIEW

**Review by sammyli0106 — APPROVED** [review]
Looks good to me.

File: frontend/src/api/index.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+const HOST = "http://localhost:3001";
```
Comment (davisgomes) [inline frontend/src/api/index.jsx:1]: Did you get this to ping the backend? Flask is listening on port 5000 and the correct pathway would be:
`http://localhost:5000/api/auth/desiredendpoint`

**Review by davisgomes — CHANGES_REQUESTED** [review]
I think everything looks good I just want to make sure you have the right pathways before merging

File: frontend/src/api/index.jsx (hunk 1 above)
Comment (zhao19991111) [reply frontend/src/api/index.jsx:1]: Okay, I will fix it

File: frontend/src/api/index.jsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+};
+
+export const signUpUser = async (
+  firstName,
+  lastName,
+  email,
+  password,
+  isTutor
+) => {
+  const res = await POST("/auth/signUp", {
```
Comment (davisgomes) [inline frontend/src/api/index.jsx:45]: will this double the forward slashes? (I.e. http://localhost:5000/api//auth/signUp). I'm not sure if that might be an issue

Comment (davisgomes) [reply frontend/src/api/index.jsx:45]: Nevermind fixed! lgtm

**Review by davisgomes — APPROVED** [review]

**Review by GeraldDT5 — APPROVED** [review]
Looks good

**Review by sammyli0106 — APPROVED** [review]
Looks good.

**Pull request merged**